airflow/executors/kubernetes_executor.py

```diff
@@ -715,6 +749,10 @@ def start(self):
         KubeResourceVersion.reset_resource_version()
         self.task_queue = self._manager.Queue()
         self.result_queue = self._manager.Queue()
+        if not self.task_queue:
+            raise AirflowException("Could not start task_queue")
```

Comment:
Is this actually possible to hit this line or is it just to make mypy happy and any excpetions would actually come from `_manager.Queue()` directly?

Reply:
Yes. It is to make mypy happy. And I think it's good.

The "root" cause of the problem is that those fields are set to None in the __init__.py. This should not be happening in the first place. But it's not easy to change now, so I could either make # type: ignore or to add the if's. 

I prefer to add if's - as the "defensive programming" practice. It's very low overhead - only at the start of the executor. And it is more of a "sanity check" than anything else.